corehq/sql_db/__init__.py

```diff
@@ -33,3 +33,32 @@ def custom_db_checks(app_configs, **kwargs):
                       'settings.{} is set to a custom database.'.format(setting))
             )
     return errors
+
+
+@register(Tags.database, deploy=True)
```

Comment:
Cool, I hadn't seen this checks framework before.  Just read up on the [docs](https://docs.djangoproject.com/en/1.11/topics/checks/), looks pretty useful.  We might want to register the check_services stuff as deploy checks too.